### ISSUE TITLE
Problem running bist FileNotFoundError: [Errno 2] No such file or directory:  (...) params.json

### ISSUE THREAD

**Issue by Catadanna** [NervanaSystems/nlp-architect#12]
I tried to run bist in your interface using the command :

`python3 server/serve.py --name bist
`

I get the following error :

```
25MB [00:28,  1.16s/MB]                                                                                                                                                                                     
Download Complete
Unzipping...
Done.
Traceback (most recent call last):
  File "serve.py", line 306, in <module>
    set_server_properties(app, args.name)
  File "serve.py", line 287, in set_server_properties
    service = Service(service_name)
  File "serve.py", line 143, in __init__
    self.service = self.load_service(service_name)
  File "serve.py", line 259, in load_service
    upload_service.load_model()
  File "/home/catalina/nlp-architect/nlp_architect/api/bist_parser_api.py", line 32, in load_model
    self.model = SpacyBISTParser()
  File "/home/catalina/nlp-architect/nlp_architect/pipelines/spacy_bist.py", line 48, in __init__
    self.bist_parser.load(bist_model if bist_model else SpacyBISTParser.pretrained)
  File "/home/catalina/nlp-architect/nlp_architect/models/bist_parser.py", line 117, in load
    with open(os.path.join(os.path.dirname(path), 'params.json'), 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/catalina/nlp-architect/nlp_architect/pipelines/bist-pretrained/params.json'
```

**Comment by lzomedia** [comment]
From what I see from the logs,  the script cannot find that params.json, have you tried creating it?

**Comment by Catadanna** [comment]
No I didn't, as I do not know what it must contain exactly.

**Comment by lzomedia** [comment]
I think is because the bust model is missing and you need to create have a
look in the python script in that path

**Comment by Catadanna** [comment]
I understand but in their documentation is written that bist is included in the package: 
http://nlp_architect.nervanasys.com/service.html

The program must have a problem.

**Comment by peteriz** [comment]
Thanks for reporting the issue.
@danielkorat can you please have a look?

**Comment by danielkorat** [comment]
The pre-trained BIST model file are not included in this repository but rather downloaded automatically on initialisation, [here](https://github.com/NervanaSystems/nlp-architect/blob/c77c9c5a570d552e055288751d9b760b94bcc90a/nlp_architect/pipelines/spacy_bist.py#L142). Can you check if it was downloaded? (The downloaded zip includes the params.json file)

**Comment by peteriz** [comment]
@Catadanna we released v0.2 yesterday with new server infrastructure. You're invited to check it out, documentation can be found [here](http://nlp_architect.nervanasys.com/service.html)
